README.md

```diff
@@ -20,6 +19,18 @@ This includes the following core Silverstripe CMS modules:
 This can be either added to an existing project or used as a project base for creating a
 basic Silverstripe CMS project.
 
+*Unlike other Silverstripe recipes, this recipe does not constrain Silverstripe core dependencies to a particular minor version line.*
+
+If you wish to use a specific minor version of Silverstripe CMS, you can specify a version of `silverstripe/framework` in your project's `composer.json` file with tilde `~`. For example:
+
+```json
+// composer.json
+// ...
+    "require": {
+        "silverstripe/framework": "~5.1",
```

Comment:
```suggestion
        "silverstripe/framework": "~5.1.0",
```
`~5.1` is the same as `^5.1` - the `.0` at the end is crucial.

Reply:
Opps

Updated